tests/node-17.0/tests.js

```diff
@@ -0,0 +1,25 @@
+const fetch = require("node-fetch");
+
+/*
+    'req' variable has:
+        'headers' - object with request headers
+        'payload' - object with request body data
+        'env' - object with environment variables
+    'res' variable has:
+        'send(text, status)' - function to return text response. Status code defaults to 200
+        'json(obj, status)' - function to return JSON response. Status code defaults to 200
+    
+    If an error is thrown, a response with code 500 will be returned.
+*/
+
+module.exports = async (req, res) => {
+    const payload = JSON.parse(req.payload);
+
+    const todo = await fetch(`https://jsonplaceholder.typicode.com/todos/${payload.id ?? 1}`).then(r => r.json());
```

Comment:
I never feel comfortable with 3rd party APIs. How important is this text? will we get the same result from just checking a simple computation?

Reply:
We do http request so we have standard library test we could do in all languages. We could use our own api, but I would strongly recommend testing runtime with a http request, because it tests if libraries work in a really simple, easy to understand and language agnostic way.

Reply:
OK 👌